osgar/tools/lidarview.py

```diff
@@ -188,9 +188,10 @@ def get_image(data):
     # https://stackoverflow.com/questions/12569452/how-to-identify-numpy-types-in-python
     if isinstance(data, np.ndarray):
         # depth data for ROBOTIKA_X2_SENSOR_CONFIG_1 (640 x 360)
+        depth_params = DepthParams()
```

Comment:
Do you have a proposal where to store the params to avoid expensive recomputation in `DepthParams.__init__()`?

Reply:
fixed - now it is using single instance of `g_depth_params`